students/sparrow925.Rmd

```diff
@@ -32,43 +32,56 @@ df
 
 ***
 
-#### <span style="color:red">**Techniques**</span>
+### <span style="color:red">**Techniques**</span>
         
 Creating code that pushes graphs reflecting up-to-date datasets without doing more than clicking a button resonates deeply with me. Bring on the pretty graphs!
 
 ***
         
-#### <span style="color:red">**Data**</span>
+### <span style="color:red">**Data**</span>
         
 So far our group is only working with data samples given in RStudio, but as we evolve this quarter I'm excited to see what we dig up!
 
 ***
-#### <span style="color:red">**Data Wrangling**</span>
+### <span style="color:red">**Data Wrangling**</span>
 
-```{r}
+Note: Unsure how to make the following chunk run without showing its results.
+```{r echo=FALSE, comment=NA}
```

Comment:
to make this chunk run without showing its results, use:
```
{r, echo=FALSE, comment=NA,  results='hide'}
```
You get these options by typing `,` after the `{r`. To name the chunk for easy navigation to it (see drop-down in lower left editor pane of Rmd file in RStudio), you'd use:
```
{r my named chunk, echo=FALSE, comment=NA,  results='hide'}
```